app/views/agents/_form.html.haml

```diff
@@ -9,7 +9,7 @@
   = hidden_field_tag :deletable, deletable if deletable
   = hidden_field_tag agent_field_name(:id, name_prefix), agent.id if agent.id
 
-  - if affiliation?(agent) &&  params[:parent_id]
```

Comment:
parent_id  is still used 
<img width="1060" alt="image" src="https://github.com/user-attachments/assets/4203f47c-9beb-430f-9fb7-e5aa8beb7f44">

I think in another PR, you changed `affiliation?(agent)` with `is_organization?(agent)`, is it not more needed?

Reply:
Here I just changed the condition cause I want to hide the radio select only when wanna create an affiliation which is always an organization and in all other cases we always leave the choice to select peron/organization.

For the is_organization naming, I'm not sure the PR I did this in it is merged or not, in case yes I think git can handle this and change it auto, else I'll do it manually

Reply:
this PR: https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/696